recipe/meta.yaml

```diff
@@ -30,12 +31,23 @@ test:
     - pip check
   requires:
     - pip
+    # packaging is unused by fastcore, but `pip check` fails 
+    # if missing it.
+    - packaging
```

Comment:
That's because `fastcore` is missing it. See here for the requirements: https://github.com/fastai/fastcore/blob/master/setup.py#L29

And here is the recipe: https://github.com/fastai/fastcore/blob/1.5.29/setup.py#L29

When a `pip check` fails, the reason is often a broken dependency. It looks like `pip check` in fastcore didn't catch that.

`fastcore` should be rebuilt or hotfixed first before merging this PR.

Reply:
See here why a discussion why packaging was left out intentionally as requested by reviewers.
https://github.com/AnacondaRecipes/fastcore-feedstock/pull/2

This should not block this PR and can be fixed by a patch in `fastcore` when it can/needs to be rebuilt

Reply:
https://github.com/AnacondaRecipes/fastcore-feedstock/pull/3/files

Reply:
Thanks for the clarification! A comment would be really helpful here, too, about why `pip check` fails